test/integration/targets/iptables/tasks/chain_management.yml

```diff
@@ -45,12 +45,6 @@
       - result is not failed
       - '"FOOBAR-CHAIN" in result.stdout'
 
-- name: flush the foobar chain
-  become: true
-  iptables:
-    chain: FOOBAR-CHAIN
-    flush: true
-
```

Comment:
Since there's no default rule to flush anymore, and this test wasn't validating state afterward anyway, it makes sense to remove this. It would be nice if there was an integration test for successfully flushing a chain though.

Reply:
Agreed. I'll get one written.